templates/jinja2/email/base_generic.html

```diff
@@ -44,7 +44,7 @@
             margin: 0;
             min-width: 100%;
             padding: 0;
-            text-align: left;
+            text-align: start;
```

Comment:
I wonder if this change is okay for emails? :thinking:

Reply:
I don't actually know how long this value has been supported. Perhaps for a long time in which case it will likely work.

I would guess that if it works with Outlook then it should be a good approach.